test/realtime/failure.test.js

```diff
@@ -185,6 +185,23 @@ define(['ably', 'shared_helper', 'async', 'chai'], function (Ably, helper, async
       }
     });
 
+    function checkIfRetryTimeoutsAreCorrect(retryTimeouts, calculationDelay = 0) {
```

Comment:
This wouldn't work on older browsers
```suggestion
    function checkIfRetryTimeoutsAreCorrect(retryTimeouts, calculationDelay) {
      calculationDelay = calculationDelay || 0;
```

Reply:
Okay, got it